github/event_types.go

```diff
@@ -653,6 +675,21 @@ type StatusEvent struct {
 	Installation *Installation     `json:"installation,omitempty"`
 }
 
+// TeamEvent is triggered when an organization's team is created, modified or deleted.
+// The Webhook event name is "team".
+//
+// Events of this type are not visible in timelines. These events are only used
+// to trigger hooks.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#teamevent
+type TeamEvent struct {
+	Action  *string       `json:"action,omitempty"`
+	Team    *Team         `json:"team,omitempty"`
+	Changes *TeamChange   `json:"changes,omitempty"`
+	Org     *Organization `json:"organization,omitempty"`
+	Sender  *User         `json:"sender,omitempty"`
```

Comment:
You should also include ``Installation *Installation     `json:"installation,omitempty"` `` here after `Sender`. It's implicitly needed for webhook payloads, according to:

> **In addition to the fields [documented for each event](https://developer.github.com/v3/activity/events/types/),** webhook payloads include the user who performed the event (`sender`) as well as the organization (`organization`) and/or repository (`repository`) which the event occurred on, and for a [GitHub App](https://developer.github.com/apps/)'s webhook **may include the installation (`installation`) which an event relates to.**

_(Source: https://developer.github.com/webhooks/#payloads.)_

Related to #505. /cc @bradleyfalzon FYI.

See events above and below for example. You can also see https://github.com/google/go-github/pull/630#discussion_r115077161 for precedent.